schema/deploy/ciip_functions/ciip_incentive_payment.sql

```diff
@@ -0,0 +1,112 @@
+-- Deploy ggircs-portal:ciip_functions/ciip_incentive_payment to pg
+-- requires: tables/benchmark
+-- requires: views/ciip_production
+-- requires: views/ciip_electricity_and_heat
+
+begin;
+
+-- Function takes application id and version number:
+create or replace function ggircs_portal.ciip_incentive_payment(app_id numeric , version_no numeric )
+returns setof ggircs_portal.ciip_incentive_by_product as $function$
+  declare
+    product record;
+    em_facility numeric;
+    em_product numeric;
+    em_electricity numeric;
+    em_heat numeric;
+    em_intensity numeric;
+    em_alloc_elec numeric;
+    em_alloc_heat numeric;
+    intensity_range numeric;
+    incentive_ratio_max numeric;
+    incentive_ratio_min numeric;
+    incentive_ratio numeric;
+    incentive_product numeric;
+    carbon_tax_facility numeric;
+    electricity_data ggircs_portal.ciip_electricity_and_heat;
+    heat_data ggircs_portal.ciip_electricity_and_heat;
+    benchmark_data ggircs_portal.benchmark;
+    product_data ggircs_portal.product;
+    product_return ggircs_portal.ciip_incentive_by_product;
+
+  begin
+     -- Define placeholder variables
+     em_alloc_elec = 0;
+     em_alloc_heat = 0;
+     incentive_ratio_min = 0;
+     incentive_ratio_max = 1;
+
+     -- Get emissions for facility
+     select sum(annual_co2e) into em_facility from ggircs_portal.ciip_emission
+     where version_number = version_no and application_id = app_id;
```

Comment:
```suggestion
     where version_number = version_no and application_id = app_id and gas_type != 'CO2';
```
biogenic emissions should not be included